apps/astarte_appengine_api/lib/astarte_appengine_api/application.ex

```diff
@@ -42,22 +38,18 @@ defmodule Astarte.AppEngine.API.Application do
     RPCConfig.validate!()
     Config.validate!()
 
-    Metrics.HealthStatus.setup()
-    Metrics.PhoenixInstrumenter.setup()
-    Metrics.PipelineInstrumenter.setup()
-    Metrics.PrometheusExporter.setup()
-    Metrics.AppEngineInstrumenter.setup()
-
     xandra_options =
       Config.xandra_options!()
       |> Keyword.put(:name, :xandra)
 
     # Define workers and child supervisors to be supervised
     children = [
-      supervisor(Astarte.RPC.AMQP.Client, []),
-      supervisor(Astarte.AppEngine.API.Rooms.MasterSupervisor, []),
-      supervisor(Astarte.AppEngine.API.Rooms.AMQPClient, []),
-      supervisor(Astarte.AppEngine.APIWeb.Endpoint, []),
+      Astarte.RPC.AMQP.Client,
+      Astarte.AppEngine.API.Rooms.MasterSupervisor,
+      Astarte.AppEngine.API.Rooms.AMQPClient,
+      Astarte.AppEngine.APIWeb.Endpoint,
+      {Phoenix.PubSub, name: Astarte.AppEngine.API.PubSub},
+      Astarte.AppEngine.APIWeb.Telemetry,
```

Comment:
And this should probably be the first thing that starts since potentially everything else can emit metrics